qt_ui/windows/settings/QSettingsWindow.py

```diff
@@ -575,6 +575,8 @@ def save_settings(self):
 
     def load_default_settings(self):
         sd = settings_dir()
+        if not sd.exists():
+            sd.mkdir()
```

Comment:
To prevent future issues regarding this topic, perhaps we should move this logic into the settings_dir function? 🤔